app/backend/core/imageshelper.py

```diff
@@ -33,11 +33,8 @@ async def download_blob_as_base64(blob_container_client: ContainerClient, file_p
         return None
 
 
-async def fetch_image(blob_container_client: ContainerClient, result: Document) -> Optional[ImageURL]:
+async def fetch_image(blob_container_client: ContainerClient, result: Document) -> Optional[str]:
     if result.sourcepage:
         img = await download_blob_as_base64(blob_container_client, result.sourcepage)
-        if img:
```

Comment:
The default is "auto", so Prompty doesn't even output detail, so we only need to return the data URI. If developers do need to customize the detail in the future, that'd require a Prompty change.